extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js

```diff
@@ -1,11 +1,22 @@
 /*global define, require*/
 define('ext.wikia.adEngine.provider.factory.wikiaGpt', [
-	'wikia.log',
 	'ext.wikia.adEngine.adLogicPageParams',
 	'ext.wikia.adEngine.provider.gpt.helper',
+	'wikia.geo',
+	'wikia.log',
 	require.optional('ext.wikia.adEngine.lookup.services')
-], function (log, adLogicPageParams, gptHelper, lookups) {
+], function (adLogicPageParams, gptHelper, geo, log, lookups) {
 	'use strict';
+	var country = geo.getCountryCode();
+
+	function overrideSizes(slotMap, newSizes) {
```

Comment:
What do you think about returning a new slot map instead of changing the passed object?